src/openrct2/drawing/ttf_sdlport.c

```diff
@@ -1261,6 +1282,146 @@ TTFSurface *TTF_RenderUTF8_Solid(TTF_Font *font,
     return textbuf;
 }
 
+TTFSurface *TTF_RenderUTF8_Shaded(TTF_Font *font,
+                const char *text, uint32 fg, uint32 bg)
+{
+    bool first;
```

Comment:
@IntelOrca Do we do variable predeclaration if the files come from an external source?

Reply:
I don't know.

Reply:
Save for @Gymnasiast's comments on formatting, I stuck to the implementation of the counterparts that were already there — which did leave the predeclaration intact.

Personally, I think it might come in handy not to refactor the SDL2_ttf code too much, if we ever want to merge changes from upstream.

Reply:
I think predeclarations are fine in some cases. This is one.